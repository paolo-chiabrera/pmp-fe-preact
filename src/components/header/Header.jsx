import { h } from 'preact';
import { Link } from 'react-router-dom';

import CryingLogo from '../crying-logo';
import MainTitle from '../main-title';

export default function Header() {
  return (
    <header>
      <Link to="/">
        <CryingLogo />
      </Link>
      <MainTitle />
    </header>
  );
}
