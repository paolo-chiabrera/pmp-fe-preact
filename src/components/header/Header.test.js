import { h } from 'preact';

import render from 'preact-render-to-string';

import { StaticRouter } from 'react-router';

import Header from './Header.jsx';

describe('Given a Header component', () => {
  it('should render correctly', () => {
    const tree = render(
      <StaticRouter location="/">
        <Header />
      </StaticRouter>
    );

    expect(tree).toMatchSnapshot();
  });
});
