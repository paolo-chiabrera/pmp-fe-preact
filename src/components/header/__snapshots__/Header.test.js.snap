// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a Header component should render correctly 1`] = `"[object Object]"`;
